pages/index.js

```diff
@@ -1,17 +1,46 @@
-import Navbar from "@/components/pages/home/Navbar";
 import Head from "next/head";
+import Navbar from "@/components/pages/home/Navbar";
 import HomeHeader from "@/components/pages/home/HomeHeader";
+import MemberCard from "@/components/pages/home/MemberCard";
 
-const Home = () => {
+const Home = ({ members }) => {
   return (
     <div>
-      <Navbar />
       <Head>
         <title>Bellshade - Learn with Our Community</title>
       </Head>
+      <Navbar />
       <HomeHeader />
+      <section
+        id="team"
+        className="max-w-[942px] m-auto"
+        style={{ width: "90%" }}
+      >
+        <div className="w-full mb-16 md:mb-20 text-center">
+          <h2 className="text-5xl md:text-[67px] font-bold">Meet Our Team</h2>
+        </div>
+        <div className="grid grid-cols-2 md:grid-cols-3 lg:grid-cols-5">
+          {members.map((member) => {
+            return <MemberCard key={member.login} {...member} />;
+          })}
+        </div>
+      </section>
```

Comment:
Yang ini nanti di pindahin ke component our teamnya

```suggestion
      <OurTeam members={members} />
```